src/Symfony/Component/PropertyAccess/Tests/StringUtilTest.php

```diff
@@ -67,6 +68,7 @@ public function singularifyProvider()
             array('emphases', array('emphas', 'emphase', 'emphasis')),
             array('faxes', 'fax'),
             array('feet', 'foot'),
+            array('feedback', 'feedback'),
```

Comment:
Hmm, I'm not sure what you mean. Feedback is not a coutable noun and by that, it can't be pluralized. See http://english.stackexchange.com/questions/176805/plural-of-feedback

The StringUtil already coped with this well, the only bug was that it was replacing all words with "ee" in it and longer than 3 characters with "oo", turing "feedback" into "foodback".
This PR adds "feedback" as an exception to this replacement statement (we may benefit from a more general fix for this, but I do not consider myself skilled enough to do that).